errors_nix.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under $license_for_repo License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !windows
+
+package python3
```

Comment:
Let's document in the README how to build for linux and windows and explain the difference.

Reply:
I don't understand. There isn't any difference. It is just a function which doesn't exist on windows

Reply:
This should be documented in the python docs not here

Reply:
Using the suffix `_nix.go` will not exclude the file from the build on windows. This means users have to specify the `-tags windows` at build time on windows to exclude this file (in order to trigger the condition with the `+build` option).

This should be specify in the readme I think.

Reply:
No the `windows` gets added automatically on windows. https://golang.org/pkg/go/build/#hdr-Build_Constraints. If you name the file `_linux.go` for example you don't even need to add the tag in the file.